RotationSolver/Updaters/TargetUpdater.cs

```diff
@@ -521,4 +524,28 @@ private static void UpdateNamePlate(IEnumerable<BattleChara> allTargets)
         }
         DataCenter.TreasureCharas = charas.ToArray();
     }
+
+    // status checks that can't easily be represented in config
+    private static bool CheckTargetableOther(BattleChara b) {
+        // Labyrinth of the Ancients
+        if (Svc.ClientState.TerritoryType == 174 && b.Name.ToString() == "Thanatos") {
+            // thanatos can only be damaged by players with Astral Realignment (398)
+            return Player.Status.Any(s => s.StatusId == 398);
+            // TODO: Allagan Bomb is also invulnerable until everything else is dead, but that's harder to check
+        }
+
+        // The Puppets' Bunker
+        if (Svc.ClientState.TerritoryType == 917) {
+            // second boss has one of three buffs corresponding to an alliance letter
+            // player needs the matching buff
+            var minibossBuff = b.StatusList.Where(x => x.StatusId == 2409 || x.StatusId == 2410 || x.StatusId == 2411).FirstOrDefault();
```

Comment:
You can just do `b.StatusList.FirstOrDefault(x => x.StatusId is 2409 or 2410 or 2411);`

I'd recommend extracting `2409`, `2410` and `2411`